db/statistics.go

```diff
@@ -625,11 +626,28 @@ func GetValidatorIncomeHistoryChart(validator_indices []uint64, currency string,
 	return clRewardsSeries, err
 }
 
-func GetValidatorIncomeHistory(validator_indices []uint64, lowerBoundDay uint64, upperBoundDay uint64, lastFinalizedEpoch uint64) ([]types.ValidatorIncomeHistory, error) {
+func GetValidatorIncomeHistory(validatorIndices []uint64, lowerBoundDay uint64, upperBoundDay uint64, lastFinalizedEpoch uint64) ([]types.ValidatorIncomeHistory, error) {
+	if len(validatorIndices) == 0 {
+		return []types.ValidatorIncomeHistory{}, nil
+	}
+
 	if upperBoundDay == 0 {
 		upperBoundDay = 65536
 	}
-	queryValidatorsArr := pq.Array(validator_indices)
+
+	validatorIndices = utils.SortedUniqueUint64(validatorIndices)
```

Comment:
As Golang doesn't support templates (afaik), I would suggest to have a `SortedUniqueUint64ToString` function as well, which do the same thing like this, just convert it to string in the append step.

Reply:
I am using the sorted uniq `validatorIndices` to create the `validatorIndicesPqArr` aswell, which I think cant handle strings.

Reply:
Pretty sure strings should be possible here, but Okay, let us keep it for now.